qiskit/synthesis/__init__.py

```diff
@@ -151,6 +151,21 @@
 .. autofunction:: multiplier_cumulative_h18
 .. autofunction:: multiplier_qft_r17
 
+Sums
+----
+
+.. autofunction:: synth_weighted_sum_carry
+
+
+Unary Arithmetic Synthesis
+==========================
+
+Integer comparators
+-------------------
+
+.. autofunction:: synth_integer_comparator_2s
+.. autofunction:: synth_integer_comparator_greedy
+
```

Comment:
the function synth_weighted_sum_carry does not appear in the docs?

Reply:
It should be there, see L157 🙂